site/docs/v1/tech/apis/identity-providers/_identity-provider-search-request-parameters.adoc

```diff
@@ -0,0 +1,34 @@
+// parameter_prefix is either blank for parameters or "search." for body
+
+[.api]
+
+[field]#{parameter_prefix}name# [type]#[String]# [optional]#Optional#::
+The case-insensitive string to search for in the Identity Provider name. This can contain wildcards using the asterisk character (`*`). If no wildcards are present, the search criteria will be interpreted as `pass:[*value*]`. If this field is omitted, no filtering will be applied to the `name` field.
+
+[field]#{parameter_prefix}numberOfResults# [type]#[Integer]# [optional]#Optional# [default]#defaults to `25`#::
+The number of results to return from the search.
+
+[field]#{parameter_prefix}orderBy# [type]#[String]# [optional]#Optional# [default]#defaults to `name ASC`#::
+The field to order the search results as well as an order direction.
++
+The possible values are:
++
+ * `id` - the UUID of the Identity Provider
+ * `insertInstant` - the link:/docs/v1/tech/reference/data-types#instants[instant] when the Identity Provider was created
+ * `name` - the Identity Provider name
+ * `enabled` - whether the identity provider is enabled
```

Comment:
What does enabled mean here? Enabled on any application?

Reply:
Each IdP configuration has a global enable toggle. Expanded on that a bit.